### PR TITLE
Fix pint issue

### PR DESCRIPTION
**Pull request by NukP**
This PR is made in respond to Issue #92 
I have remove the definition of Refractive index from dgpost as suggested.
I have tested this and it now works just fine (can now process the data like before)

### CODE REVIEW

File: tests/__init__.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
 import pytest
 
 pytest.register_assert_rewrite("tests.utils")
+
+#Random comment
```
Comment (PeterKraus) [inline tests/__init__.py:5]: Please revert those changes.

File: tests/test_catalysis_atbal.py [hunk 2]
```diff
@@ -106,4 +106,4 @@ def test_catalysis_atbal_rinxin(datadir):
     assert np.allclose(df["C1"], pd.DataFrame([1.0, 1.0, 0.995, 1.005, 1.01, 0.99]))
     assert np.allclose(df["C2"], pd.DataFrame([1.0, 1.0, 0.995, 1.005, 1.01, 0.99]))
     assert np.allclose(df["O1"], pd.DataFrame([1.0, 1.0, 0.95, 1.05, 1.0, 1.0]))
-    assert np.allclose(df["O2"], pd.DataFrame([1.0, 1.0, 0.95, 1.05, 1.0, 1.0]))
+    assert np.allclose(df["O2"], pd.DataFrame([1.0, 1.0, 0.95, 1.05, 1.0, 1.0]))
```
Comment (PeterKraus) [inline tests/test_catalysis_atbal.py:109]: Please revert those changes, too.

**Review by PeterKraus — CHANGES_REQUESTED** [review]

**Comment by NukP** [comment]
I have rebased the brach so that those commits have been reverted from this PR as requested.

**Review by PeterKraus — APPROVED** [review]
I bumped the minimum version of `pint` to `>=0.24.1`. The RIU were introduced into `pint` in `0.24`, but that one is not compatible with `python==3.9`, so it's better to ask for `0.24.1` and above.

**Pull request merged**